### DISCUSSION TITLE
How to do simple search vector for text (No Neural)

### DISCUSSION THREAD

**Discussion by roysG** [qdrant/qdrant#3281] [General]
Hi,
Please give some example how to do simple search vector for text.
Thanks

**Comment by generall** [comment]
Hi @roysG,

You can use full-text filter - https://qdrant.tech/documentation/concepts/filtering/#full-text-match

in combination with scroll request: https://qdrant.tech/documentation/concepts/points/#scroll-points

**Comment by roysG** [comment]
In case i 1,000,000 of items, 
Is this will search as vector (fast search) or payload (slow search, required defined index) ?

**Comment by timvisee** [comment]
If using payload search, then yes, it is highly recommended to define an index on the payload key you're filtering.

**Comment by roysG** [comment]
I want to do search on text but with vector search.

When I check all your examples, I see that you talk only about semantic
search.

I want a regular search but with vector

Do you understand?

בתאריך יום ג׳, 31 באוק׳ 2023 ב-12:11 מאת Tim Visée ***@***.***

**Comment by generall** [comment]
> Do you understand?

Not really.

Could you please describe an example of regular, non-semantic, search but with vector?

**Comment by roysG** [comment]
Hi Yes,
For example,

In case i will write the word:  car
and search for the word vehicle,

Then "vehicle"  will get high score  in the semantic way for example:  all-MiniLM-L6-v2 , 
cause it is the same meaning  in other word.

As you can understand  car and  vehicle are different words.
I need to do text match but in vector way.

So, when i will search the word vehicle i will get low score, cause its content charcters are difference than car

I hope i explained myself

**Comment by generall** [comment]
Why do you call it `regular search but with vector`? In my book it is a definition of semantic search

**Comment by roysG** [comment]
Yes, 
But in case i want to see all the car words just if i type car.. then i want to get car
But, if i typed vehicle then i do not need to get the car strings..
I should get string of vehicle

For example lets take 3 strings:
car is good
car is big
vehicle is great

In case i will search car.. then the last item in the example strings should not seen, or seen with very low score.
In case i will search vehicle then the car should be with with low score.

In the semantic search, the vehicle will get at least 0.6,
I am looking for solution that the score will be low score, cause the word is different than car.
